types/react-mathquill/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for react-mathquill 0.1
+// Project: https://github.com/viktorstrate/react-mathquill#readme
+// Definitions by: Marco Gonzalez <https://github.com/magonzalez9>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import { ComponentProps, ReactNode, Component } from 'react';
+
+export interface MathField {
```

Comment:
In the source code, this type is part of the [`@edtr-io/mathquill`] library. I propose to separate all types that relate to this library.

[`@edtr-io/mathquill`]: https://www.npmjs.com/package/@edtr-io/mathquill

Reply:
So create a new pull request for the mathquill library?

Reply:
Why not in the current?

Reply:
How do you suggest I do that?

Reply:
Create a subfolder with an index.d.ts file for @edtr-io/mathquill?

Reply:
[Documentation](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#what-about-scoped-packages) have example.

Reply:
> Why not in the current?

I meant this:

> Why not in the current Pull Request?

Reply:
Done

Reply:
Do you know how long until this get re-reviewed?